api/v1/kataconfig_types.go

```diff
@@ -70,10 +70,12 @@ type KataConfigStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=kataconfigs,scope=Cluster
-// +kubebuilder:printcolumn:name="InProgress",type=string,JSONPath=".status.installationStatus.IsInProgress",description="Status of Kata runtime installation"
-// +kubebuilder:printcolumn:name="Completed",type=integer,JSONPath=".status.installationStatus.completed.completedNodesCount",description="Number of nodes with Kata runtime installed"
+// +kubebuilder:printcolumn:name="InstallInProgress",type=string,JSONPath=".status.installationStatus.IsInProgress",description="Status of Kata runtime installation"
```

Comment:
Long back @gkurz suggested to try this but I never got to try a conditional - `status.installationStatus.IsInProgress | status.unInstallationStatus.inProgress.status`
Can you try this ?

Reply:
Verified this does not work. With patch applied, the output is as follows:

```
$ oc get kataconfig
NAME
example-kataconfig

```

Reply:
> I think I like it better, but that's a personal taste :-)
> 

So, I think we could proceed with the proposed patch, and I will work on trying to improve the output, as it requires some investigation. I don't really know how properly do it yet.

> I usually look at "oc describe" to get the status of uninstallation, but it's definitely useful to have this status on the "get" command too.

I will check the output with oc describe, thanks. I still need some more fluency on the `oc` commands.

Reply:
> > I usually look at "oc describe" to get the status of uninstallation, but it's definitely useful to have this status on the "get" command too.
> 
> I will check the output with oc describe, thanks. I still need some more fluency on the `oc` commands.

In the end,  `oc describe` is equivalent to the YAML view in the web interface. TIL.